packages/@magic-sdk/react-native-expo/src/react-native-webview-controller.tsx

```diff
@@ -216,7 +229,18 @@ export class ReactNativeWebViewController extends ViewController {
   }
 
   protected async _post(data: any) {
+    const methodType = data.payload.method;
+    const messageId = data.payload.id;
+
     if (this.webView && (this.webView as any).postMessage) {
+      // Setup timeout for message response
+      const timeout = setTimeout(() => {
+        this.messageTimeouts.delete(messageId);
+        throw createResponseTimeoutError(`Response timed out for method: ${methodType} with message id: ${messageId}`);
```

Comment:
nit: I'd prefer we pass `methodType` and `messageId` as arguments to `createResponseTimeoutError ` and create the error message inside that function, as it sounds like it would be the same everywhere.